include/wx/persist/splitter.h

```diff
@@ -23,6 +23,9 @@
 // Special position value of -1 means the splitter is not split at all.
 #define wxPERSIST_SPLITTER_POSITION wxASCII_STR("Position")
 
+#define wxPERSIST_SPLITTER_DEFAULT_HORIZONTAL wxASCII_STR("Horizontal")
+#define wxPERSIST_SPLITTER_DEFAULT_VERTICAL wxASCII_STR("Vertical")
```

Comment:
To be consistent with the naming schema suggested above I'd also use
```suggestion
#define wxPERSIST_SPLITTER_DEFAULT_HORIZONTAL wxASCII_STR("LastHorz")
#define wxPERSIST_SPLITTER_DEFAULT_VERTICAL wxASCII_STR("LastVert")
```
here.